server/models/Notification.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require("mongoose");
+
+const notificationSchema = new mongoose.Schema({
+    noti_type: {
+        type: String,
+        required: true
+    },
+    receiver: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: "User",
+        require:true
+    },
+    content:{
+        title: String, 
+        image: String, 
+        description: String,
+        url: String,
+        isRead: {
+            type: Boolean,
+            default: false
+        },
+        product_id: {
```

Comment:
We should stick to a naming style. JavaScript convention is camelcase 'productId'. Python is snake case product_id.

Reply:
Yes, my mistake, I will correct them on my current branch.